src/Products/CMFCore/DirectoryView.py

```diff
@@ -359,10 +359,11 @@ def listDirectories(self):
 registerMetaType = _dirreg.registerMetaType
 
 
-def listFolderHierarchy(ob, path, rval, adding_meta_type=None):
+def listFolderHierarchy(ob, path, rval, adding_meta_type=None, max=0, c=0):
```

Comment:
Please use meaningful parameter names, not just `c`

Reply:
Thanks for the hint. Done: https://github.com/zopefoundation/Products.CMFCore/pull/130/commits/823ba67709f6391911f51adc315e9a275e371861